src/cli/mod.rs

```diff
@@ -42,8 +45,6 @@ pub fn parse_cli_args() -> (String, Config) {
     );
     trace!("Program was called with CLI 😉");
     trace!("Parsed the arguments");
-    debug!("{:?}", opts.text);
```

Comment:
Why get rid of these debug messages?

Reply:
We will force people to tell us their CLI arguments if they want help, and these two are already printed elsewhere in the program (text == Athena, verbosity == you will see it if there are debug / info / trace logs)

Reply:
Alright.